python/graphstorm/config/argument.py

```diff
@@ -537,6 +538,18 @@ def verbose(self):
 
         return False
 
+    @property
+    def use_wholegrap(self):
```

Comment:
How about use_wholegraph_embed?
As use_wholegraph for feature store is automatically enabled if features are stored in wholegraph format, If we use `use_wholegraph` here, it is a little bit confusing.

Reply:
Sounds good. I will update it accordingly